docs/docker-integration/cloudformation.rst

```diff
@@ -61,27 +61,78 @@ Once it's up and running we'll guide you through a tutorial to deploy a sample a
             <br />
 			This button will open CloudFormation in a new window.</p>
 		</div>
+
 		<div class="step-stages__step first">
+
+            <img src="../_images/10-cloudformation-firstpage.png" alt="Initial CloudFormation page" />
+			<span>Click "Next" to proceed past the first page.</span>
+
+
+		</div>
+		<div class="step-stages__step">
+
 			<img src="../_images/11-cloudformation-stackname.png" alt="Specifying the stack name"/>
-			<span>Click "Next"</span>
-			<span>Enter a <code>Stack name</code>. This can be any descriptive name.</span>
-			<span>Enter your <code>KeyPair</code> name from Step 1. Then enter your AWS <code>AccessKeyID</code> and <code>SecretAccessKey</code> credentials.</span><span>If you don't know these, <a href="javascript:void(0);" onclick="$('#iam-instructions').show();">click here</a>.</span>
+			<span>
+            <ul>
+                <li>
+                    Enter a <code>Stack name</code>. This can be any descriptive name.
+                </li>
+                <li>
+                    Enter your <code>KeyPair</code> name from Step 1. Then enter your AWS <code>AccessKeyID</code> and <code>SecretAccessKey</code> credentials.</span><span>If you don't know these, <a href="javascript:void(0);" onclick="$('#iam-instructions').show();">click here</a>.
```

Comment:
:question: I'm told that "click here" is bad practice. Perhaps just put "Don't know your AWS credentials?" in the href?